### DISCUSSION TITLE
Custom script for level designers

### DISCUSSION THREAD

**Discussion by valkyrienyanko** [Valks-Games/sankari#153] [Ideas]
![image](https://user-images.githubusercontent.com/6277739/188399442-84832246-9e7a-430c-8402-a3c0a2b5ebcf.png)

![image](https://user-images.githubusercontent.com/6277739/188399483-1568a196-e731-4b16-9f11-f9c8e9fb02f7.png)

**Comment by Soroosh-Sharif** [comment]
It seems relevant to #49. I was going to create a plugin that helps with coin placement, but it can be extended to be a full level designer. Let me create a very simple coin placer plugin (#49) so you can see if this is the direction you wanna go.
